uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Reminder.php

```diff
@@ -588,7 +588,7 @@ private static function _getContactDailyReminderData($contactId, array $activity
 
         // Fill the $reminderData array:
         if ($reminderKey) {
-          $isTask = isset($activityResult->subject);
+          $isTask = self::_checkIfActivityTypeBelongsToComponent($activityResult->activity_type_id, 'CiviTask');
```

Comment:
This function name is a bit long here.
What about creating a wrapper function
`isTaskComponent` that accepts the activity type_id and uses the `checkIfActivityTypeBelongsToComponent` function internally.

```php
function isTaskComponent($activityTypeId) {
return self::_checkIfActivityTypeBelongsToComponent($activityTypeId, 'CiviTask')
}
```
Would be more readable and easier to comprehend.